x-pack/plugins/security_solution/public/detections/components/alerts_table/grouped_alerts.tsx

```diff
@@ -238,30 +236,25 @@ export const GroupedAlertsTableComponent: React.FC<AlertsTableComponentProps> =
 
   const groupedAlerts = useMemo(
     () =>
-      isNoneGroup(selectedGroup) ? (
-        renderChildComponent([])
-      ) : (
-        <GroupingContainer
-          badgeMetricStats={(fieldBucket: RawBucket) =>
-            getSelectedGroupBadgeMetrics(selectedGroup, fieldBucket)
-          }
-          customMetricStats={(fieldBucket: RawBucket) =>
-            getSelectedGroupCustomMetrics(selectedGroup, fieldBucket)
-          }
-          data={alertsGroupsData?.aggregations ?? {}}
-          groupPanelRenderer={(fieldBucket: RawBucket) =>
-            getSelectedGroupButtonContent(selectedGroup, fieldBucket)
-          }
-          groupsSelector={groupsSelector}
-          inspectButton={inspect}
-          isLoading={loading || isLoadingGroups}
-          pagination={pagination}
-          renderChildComponent={renderChildComponent}
-          selectedGroup={selectedGroup}
-          takeActionItems={getTakeActionItems}
-          unit={defaultUnit}
-        />
-      ),
+      isNoneGroup(selectedGroup)
+        ? renderChildComponent([])
+        : getGroupedTables({
+            badgeMetricStats: (fieldBucket: RawBucket) =>
+              getSelectedGroupBadgeMetrics(selectedGroup, fieldBucket),
+            customMetricStats: (fieldBucket: RawBucket) =>
+              getSelectedGroupCustomMetrics(selectedGroup, fieldBucket),
+            data: alertsGroupsData?.aggregations ?? {},
```

Comment:
When we change interface to `data?: GroupingTableAggregation & GroupingFieldTotalAggregation;` that will simplify the code here to `data: alertsGroupsData?.aggregations`

Reply:
done!